### DISCUSSION TITLE
Fix examples README?

### DISCUSSION THREAD

**Discussion by cottrell** [paveldedik/ludic#14] [Q&A] [answered]
```
ludic/examples$ uvicorn bulk_update:app --reload
```

This is the instruction but this does not run. Put a complete working example including run path in the README

**Comment by paveldedik** [comment]
Thank you for testing it. Does this work for you?

```python
cd examples/
PYTHONPATH="${PYTHONPATH}:$(pwd)/.." uvicorn bulk_update:app --reload
```

**Reply by paveldedik** [reply, marked as answer]
You can also go to the root of the repository and run:

```bash
uvicorn examples.bulk_update:app
```

Which is probably better and I change that in the README probably.

**Reply by cottrell** [reply]
Yep

```
ludic$ uvicorn examples.bulk_update:app
```

works
